hoomd/hpmc/integrate.py

```diff
@@ -957,9 +964,12 @@ def format_param_pos(self, param):
         return shape_def
 
     def get_type_shapes(self):
-        """ Get all the types of shapes in the current simulation
+        """Get all the types of shapes in the current simulation.
+
+        Currently assumes that all 3D shapes are convex.
```

Comment:
Are these notices necessary for 2D shape classes?

Reply:
Nope - removing now.

Reply:
For now, I removed all the notices about 3D shapes and assumed convexity in HPMC. The `get_type_shapes` function is only currently implemented for convex shapes, which ensures this is always true. In `dem/pair.py`, I added the warning about assumed convexity because DEM specifies both vertex positions and face indices in its shape parameters (though I'm unsure if it actually works with non-convex shapes).